Release 214.0.0 (#4762)
MetaMask/core: packages/assets-controllers/CHANGELOG.md

## [Unreleased]

## [38.2.0]

### Changed

> - Fetch token rates for all accounts instead of only the selected account ([#4759](https://github.com/MetaMask/core/pull/4759))

Reviewer: It's also worth noting that this change specifically applies to the `TokenRatesController`.
```suggestion
- The TokenRatesController now fetches token rates for all accounts, instead of just the selected account ([#4759](https://github.com/MetaMask/core/pull/4759))
```

> - Fetch token rates across all accounts instead of just the selected account ([#4759](https://github.com/MetaMask/core/pull/4759))

Reviewer: Nit:
```suggestion
Fetch token rates for all accounts instead of only the selected account ([#4759](https://github.com/MetaMask/core/pull/4759))
```

Author: Thanks! updates 🙏
